src/test/groovy/hudson/plugins/gradle/injection/BuildScanInjectionMavenIntegrationTest.groovy

```diff
@@ -162,6 +162,9 @@ class BuildScanInjectionMavenIntegrationTest extends BaseMavenIntegrationTest {
             with(it.next()) {
                 it == '-Dgradle.enterprise.url=https://scans.gradle.com'
             }
+            with(it.next()) {
+                it == '-Dgradle.scan.captureGoalInputFiles=false'
```

Comment:
🤔 Shouldn't it be `gradle.scan.captureGoalInputFiles=true` by default ?

Reply:
In this specific test, the `turnOnBuildInjectionAndRestart` method sets it to `false` by default.
I've changed the default value to `true`